AdyenUIHostUITests/Components/AtomeComponentUITests.swift

```diff
@@ -0,0 +1,212 @@
+        postalCodeTextfield.tap()
+        postalCodeTextfield.typeText("1089")
+
+        app.otherElements.buttons[AtomeViewIdentifier.payButton].tap()
+
+        wait(for: .seconds(5))
+
+        if app.webViews.buttons.containing(NSPredicate(format: "label BEGINSWITH 'Pay in Browser'")).element.exists {
+            app.webViews.buttons.containing(NSPredicate(format: "label BEGINSWITH 'Pay in Browser'")).element.tap()
+            
+            XCTAssertTrue(app.webViews.textFields[AtomeViewIdentifier.inputOTP].exists)
+
+            app.otherElements.webViews.textFields[AtomeViewIdentifier.inputOTP].tap()
+            app.otherElements.webViews.textFields[AtomeViewIdentifier.inputOTP].typeText("3153")
+            
+            XCTAssertTrue(app.webViews.buttons.containing(NSPredicate(format: "label BEGINSWITH 'Next'")).element.exists)
+
+            app.webViews.buttons.containing(NSPredicate(format: "label BEGINSWITH 'Next'")).element.tap()
+            
+            wait(for: .seconds(2))
+
+            let confirmButton = app.webViews.buttons.containing(NSPredicate(format: "label BEGINSWITH 'Confirm payment'"))
+            if confirmButton.element.exists {
+                confirmButton.element.tap()
+            }
+            wait(for: .seconds(2))
+
+            app.swipeUp()
+
+            app.webViews.buttons.containing(NSPredicate(format: "label BEGINSWITH 'Back to merchant'")).element.tap()
+            app.alerts["Authorised"].scrollViews.otherElements.buttons["OK"].tap()
+        }
+    }
+
+    func testCompleteAtomeflow_through_loginRegster_shouldsuccess() {
+        let firstNameTextfield = app.otherElements.textFields[AtomeViewIdentifier.firstName]
+        firstNameTextfield.tap()
+        firstNameTextfield.typeText("John")
+        
+        let lastNameTextfield = app.otherElements.textFields[AtomeViewIdentifier.lastName]
+        lastNameTextfield.tap()
+        lastNameTextfield.typeText("Smith")
+
+        let phoneNumberTextfield = app.otherElements.textFields[AtomeViewIdentifier.phone]
+        phoneNumberTextfield.tap()
+        phoneNumberTextfield.typeText("80002018")
+
+        let streetTextfield = app.otherElements.textFields[AtomeViewIdentifier.streetName]
+        streetTextfield.tap()
+        streetTextfield.typeText("North-Ridge")
+
+        let apartmentTextfield = app.otherElements.textFields[AtomeViewIdentifier.apartmentName]
+        apartmentTextfield.tap()
+        apartmentTextfield.typeText("FlowerStreet Bell")
+
+        let houseNumberTextfield = app.otherElements.textFields[AtomeViewIdentifier.houseNumberOrName]
+        houseNumberTextfield.tap()
+        houseNumberTextfield.typeText("345")
+
+        let postalCodeTextfield = app.otherElements.textFields[AtomeViewIdentifier.postalCode]
+        postalCodeTextfield.tap()
+        postalCodeTextfield.typeText("1089")
+
+        app.otherElements.buttons[AtomeViewIdentifier.payButton].tap()
+
+        wait(for: .seconds(5))
```

Comment:
These durations (5 sec, 2 sec) seem very long but I don't know much about UI tests. Are they needed to be this much in UI tests?